lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -296,6 +297,16 @@ class ReportUIFeatures {
     });
   }
 
+  _setupElementScreenshotOverlay() {
+    const json = this.json;
+    const fullPageScreenshot = /** @type {LH.Artifacts['FullPageScreenshot'] | undefined} */ (
```

Comment:
switch to the audit details type now?

(And may be able to get away with getting `details` and testing the `type` property to make tsc happy without the casting)